tidb-binlog/pump_client/client_test.go

```diff
@@ -14,6 +14,7 @@
 package client
 
 import (
+	"context"
```

Comment:
what？

Reply:
yes. ???

Reply:
sure you can change it, it's same type go>=1.9(by type aliases)

Reply:
all use context